spec/models/scenario/year_interpolator_spec.rb

```diff
@@ -94,6 +94,35 @@
     end
   end
 
+  context 'with a scenario containing a boolean input' do
+    let(:source) do
+      FactoryBot.create(:scenario, {
+        id: 99999,
+        end_year: 2050,
+        user_values: {
+          'grouped_input_one' => 75,
+          'bool' => 0
```

Comment:
This test would be more useful when we test for a future value of 1, given that the present value is always 0. Then we can see whether the interpolation will keep the value at 1 for the intermediate scenario.